config-linux.md

```diff
@@ -336,6 +336,11 @@ The following properties do not specify memory limits, but are covered by the `m
     To disable it, specify a value of `true`.
 * **`useHierarchy`** *(bool, OPTIONAL)* - enables or disables hierarchical memory accounting.
     If enabled (`true`), child cgroups will share the memory limits of this cgroup.
+* **`checkBeforeUpdate`** *(bool, OPTIONAL)* - enables container memory usage check before setting a new limit.
```

Comment:
could we make it clearer that it is about the memory limit?  Perhaps `checkMemoryUsageBeforeUpdate`?

Reply:
Well, this is already under `.linux.memory.checkBeforeUpdate` so adding a second "memory" would be a tautology.